public/data/Lavakush Biyani.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "Lavakush Biyani",
+  "type": "personal",
+  "bio": "I am Lavakush Biyani, currently pursuing my B.tech. I am a tech enthusiast & an open-source contributor. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/lavakush07.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/lavakush07",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/lavkushbiyani1",
+      "icon": "twitter"
+    },
+    {
+     "name":"Follow me on Linkedin",
```

I think we should capitalize the `i` 😅 What do you think? 
```suggestion
     "name":"Follow me on LinkedIn",
```

---

Haha 😀, Do I  commit the changes u made it ? I am in confusion 😅

---

> Haha 😀, Do I commit the changes u made it ? I am in confusion 😅

Sure if you want, except for the resolved conversation. :)